src/main/java/ua/danit/final_project/repositories/TaskRepository.java

```diff
@@ -4,6 +4,16 @@
 import org.springframework.stereotype.Repository;
 import ua.danit.final_project.entities.Task;
 
+import java.util.Arrays;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Arrays'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=109&issues=AWXI83-b_eEZPtUmp6G0&open=AWXI83-b_eEZPtUmp6G0)

[ISSUE_KEY]: AWXI83-b_eEZPtUmp6G0